### ISSUE TITLE
StringIndexOutOfBoundsException when detecting module relative path

### ISSUE THREAD

**Issue by alessiom72** [wt-io-it/odoo-pycharm-plugin#58]
Plugin Version: 0.6.0
IntelliJ Version: IntelliJ IDEA 2022.2.3
IntelliJ Build: IU-222.4345.14

<details><summary>java.lang.StringIndexOutOfBoundsException: begin 48, end 47, length 47</summary>

```
java.lang.StringIndexOutOfBoundsException: begin 48, end 47, length 47
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4604)
	...
	at java.base/java.lang.String.substring(String.java:2680)
	at at.wtioit.intellij.plugins.odoo.modules.impl.AbstractOdooModuleImpl.getRelativeLocationString(AbstractOdooModuleImpl.java:45)
	at at.wtioit.intellij.plugins.odoo.AbstractOdooCompletionContributor.lambda$suggestModelName$0(AbstractOdooCompletionContributor.java:85)
	at at.wtioit.intellij.plugins.odoo.WithinProject.run(WithinProject.java:25)
	at at.wtioit.intellij.plugins.odoo.AbstractOdooCompletionContributor.suggestModelName(AbstractOdooCompletionContributor.java:79)
	at at.wtioit.intellij.plugins.odoo.OdooXmlCompletionContributor.fillCompletionVariants(OdooXmlCompletionContributor.java:16)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:76)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:297)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
thanks @alessiom72 for reporting this, it's fixed in 0.6.2 (pending jetbrains approval). if you are eager to test it you can download the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.2

**Comment by bigbear3001** [comment]
0.6.2 has been approved and is now available in the marketplace
